airbyte-integrations/connectors/source-opsgenie/source_opsgenie/schemas/alerts.json

```diff
@@ -0,0 +1,143 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": ["null", "string"]
+    },
+    "tinyId": {
+      "type": ["null", "string"]
+    },
+    "alias": {
+      "type": ["null", "string"]
+    },
+    "message": {
+      "type": ["null", "string"]
+    },
+    "status": {
+      "type": ["null", "string"]
+    },
+    "acknowledged": {
+      "type": ["null", "boolean"]
+    },
+    "seen": {
+      "type": ["null", "boolean"]
+    },
+    "isSeen": {
+      "type": ["null", "boolean"]
+    },
+    "tags": {
+      "type": ["null", "array"],
+      "items": {
+        "type": "string"
+      }
+    },
+    "snoozed": {
+      "type": ["null", "boolean"]
+    },
+    "snoozedUntil": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "count": {
+      "type": ["null", "number"]
+    },
+    "lastOccurredAt": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "createdAt": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "updatedAt": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "source": {
+      "type": ["null", "string"]
+    },
+    "owner": {
+      "type": ["null", "string"]
+    },
+    "priority": {
+      "type": ["null", "string"]
+    },
+    "responders": {
+      "type": ["null", "array"],
+      "items": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": ["null", "string"]
+          },
+          "type": {
+            "type": ["null", "string"]
+          }
+        }
+      }
+    },
+    "teams": {
+      "type": ["null", "array"],
+      "items": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": ["null", "string"]
+          }
+        }
+      }
+    },
+    "integration": {
+      "type": ["null", "object"],
+      "properties": {
+        "id": {
+          "type": ["null", "string"]
+        },
+        "name": {
+          "type": ["null", "string"]
+        },
+        "type": {
+          "type": ["null", "string"]
+        }
+      }
+    },
+    "report": {
+      "type": ["null", "object"],
+      "properties": {
+        "ackTime": {
+          "type": ["null", "number"]
+        },
+        "closeTime": {
+          "type": ["null", "number"]
+        },
+        "acknowledgedBy": {
+          "type": ["null", "string"]
+        },
+        "closedBy": {
+          "type": ["null", "string"]
+        }
+      }
+    },
+    "actions": {
+      "type": ["null", "array"],
+      "items": {
+        "type": "string"
+      }
+    },
+    "entity": {
+      "type": ["null", "string"]
+    },
+    "description": {
+      "type": ["null", "string"]
+    },
+    "details": {
+      "type": ["null", "object"],
+      "properties": { },
+      "additionalProperties": true
+    },
+    "ownerTeamId": {
```

Comment:
@adamschmidt I can’t find `ownerTeamId` in the API docs, you have the right schemas identified? https://docs.opsgenie.com/docs/alert-api

Reply:
This was based on live data from the account I'm using for dev/testing. Perhaps it's an undocumented field?

I can confirm that it's being consistently populated in the data I'm retrieving in testing.